### ISSUE TITLE
Support for custom CA certs

### ISSUE THREAD

**Issue by tabrezm** [deedy5/primp#31]
I'm using primp with a proxy and need to be able to provide my own CA certs.  Currently using `verify=False` as a workaround. Can you consider this feature request? Thanks

**Comment by tabrezm** [comment]
@deedy5 your reply [here](https://github.com/deedy5/primp/issues/13#issuecomment-2272067775) made me realized there might be a workaround where I copy/install my custom CA cert in a specific location and it will automatically be used by primp/rquest. Can you share some tips on that if it's an option?

**Comment by deedy5** [comment]
@tabrezm What location did you paste the certificate in?

**Comment by deedy5** [comment]
Implemented in v0.6.0

```python
import certifi
import primp

resp = primp.Client(impersonate="safari_17.5", ca_cert_file=certifi.where()).get(
    "https://foxnews.com"
)
print(resp.text_rich)
```
